Teradata/vantage-ui-platform
# feat(sqle): add sqle package

Test steps

- `npm run build:lib`
- `npm pack ./deploy/ui-platform/`
- Go into another repo 😉  and update the package json to `@td-vantage/ui-platform": "file:../vantage-ui-platform/td-vantage-ui-platform-1.0.0-beta.0.tgz", `(Change path if necessary)
- `npm i` within this new repo
- Pull in module and verify services work as expected
